server/middlewares/build-page.js

```diff
@@ -42,7 +42,7 @@ function getCommonData(pagePath, i18n) {
     var pageName = path.basename(pagePath);
 
     return {
-        assetsPath: hosts.static + pagePath + '/_' + pageName,
+        assetsPath: hosts.static + pagePath + '/' + pageName,
```

Comment:
Hmm, in the debianizer we copy only files prefixed with underscore from the build directory. So it's kind of a semantic mark meaning "build artifact".
But that itself is another relic itself - we have human generated files in the build directory which is wrong.